### DISCUSSION TITLE
PROMO Code

### DISCUSSION THREAD

**Discussion by ahmed4629** [Rehankhalil462/Alan-AI-Studio-Source-Code#6] [Q&A] [answered]
Can you provide free promo code for free alan ai credits so i can test and use during development?

**Comment by Rehankhalil462** [comment, marked as answer]
Yes . 
'CODEVO' use this promo code to get 1500 free interactions. Thanks me later.

**Reply by ahmed4629** [reply]
Thanks . It works !

**Reply by shaikh-abdulvasim** [reply]
but its not working now  please give me another code i urgently need one for my final year project
